app/services/usps_in_person_proofing/mock.rb

```diff
@@ -1,26 +1,36 @@
+require './spec/support/usps_ipp_fixtures'
```

Comment:
It's weird to import a spec file into the application code but I wanted to use the fixture-loading logic. Thoughts on this?

Reply:
I left a related comment at https://github.com/18F/identity-idp/pull/6629#discussion_r944453452